R/asis-nodes.R

```diff
@@ -64,7 +64,7 @@ inline_dollars_regex <- function(type = c("start", "stop", "full")) {
   switch(type,
     start = start,
     stop = stop,
-    full = glue::glue('({start}.+?{stop})')
+    full = glue::glue('({start}.*?{stop})')
```

Comment:
OHHH I see what's happening here 👀 😮 

The `{start}` is actually including the first character because it matches any non-punctuation or space after the delimiter, so it must match a math symbol with a single character. Good catch 💯